### ISSUE TITLE
Deploy BP nodes with Isaac's PCB patch

### ISSUE THREAD

**Issue by trevorbernard** [Granola-Team/infra#18]

**Comment by robinbb** [comment]
Blocked by #696

**Comment by jhult** [comment]
Closed as a duplicate of https://github.com/Granola-Team/mina-indexer/issues/476

**Comment by robinbb** [comment]
Moving this to Equivalence milestone because it seems that Gareth is still running his Mina node. (This makes sense, because he must still do block production on the hard-forked chain.)

**Comment by n1tranquilla** [comment]
Can we use his node for now and call ourselves equivalent on this matter?
